checkov/common/util/file_utils.py

```diff
@@ -8,6 +9,12 @@ def convert_to_unix_path(path: str) -> str:
     return path.replace('\\', '/')
 
 
+def extract_tar_archive(source_path: str, dest_path: str) -> None:
+    tar = tarfile.open(source_path)
+    tar.extractall(path=dest_path)
+    tar.close()
```

Comment:
```suggestion
    with tarfile.open(source_path) as tar:
        tar.extractall(path=dest_path)
```
always better to use a context manager 😉